src/CallHandler.tsx

```diff
@@ -729,18 +727,25 @@ export default class CallHandler extends EventEmitter {
 
         Modal.createTrackedDialog('Media capture failed', '', ErrorDialog, {
             title, description,
-        }, null, true);
+        }, undefined, true);
     }
 
-    private async placeCall(roomId: string, type: PlaceCallType, transferee: MatrixCall) {
+    private async placeCall(roomId: string, type: PlaceCallType, transferee?: MatrixCall) {
         Analytics.trackEvent('voip', 'placeCall', 'type', type);
         CountlyAnalytics.instance.trackStartCall(roomId, type === PlaceCallType.Video, false);
 
         const mappedRoomId = (await VoipUserMapper.sharedInstance().getOrCreateVirtualRoomForRoom(roomId)) || roomId;
         logger.debug("Mapped real room " + roomId + " to room ID " + mappedRoomId);
 
-        const timeUntilTurnCresExpire = MatrixClientPeg.get().getTurnServersExpiry() - Date.now();
-        console.log("Current turn creds expire in " + timeUntilTurnCresExpire + " ms");
+        const turnServersExpiry = MatrixClientPeg.get().getTurnServersExpiry();
+        if (turnServersExpiry) {
+            const timeUntilTurnCresExpire = turnServersExpiry - Date.now();
+            console.log("Current turn creds expire in " + timeUntilTurnCresExpire + " ms");
+        } else {
+            if (process.env.NODE_ENV === 'development') {
```

Comment:
This isn't really growing on me. I'd rather get loglevel in to do things like this, or wrap it up in an `assert()` that's a nop in production mode or something.

Moreover, in this case, I forget how this works exactly, but what happens when the client can't get turn creds (in which case we should be attempting the call anyway).

Reply:
In some cases it just looked like we aren't properly checking for failures and I honestly have no idea what should happen here. This seemed like the best way to do it without breaking current experience, but it doesn't sound correct.